build_tools/github_actions/write_torch_versions.py

```diff
@@ -91,13 +87,23 @@ def get_all_wheel_versions(
 
 
 def main(argv: list[str]):
+    env_dist_dir = os.getenv("PACKAGE_DIST_DIR")
     p = argparse.ArgumentParser(prog="write_torch_versions.py")
     p.add_argument(
         "--dist-dir",
         type=Path,
-        default=Path(os.getenv("PACKAGE_DIST_DIR")),
+        default=Path(env_dist_dir if not env_dist_dir == None else "<no-valid-dir>"),
         help="Path where wheels are located",
     )
+
+    if args.dist_dir == "<no-valid-dir>":
```

Comment:
I don't think this check will ever succeed as written, since `args.dist_dir` is a `Path` object:
```python
>>> from pathlib import Path
>>> path = Path("<no-valid-dir>")
>>> path == "<no-valid-dir>"
False
```

```suggestion
        default=Path(env_dist_dir if not env_dist_dir == None else "<no-valid-dir>"),
help="Path where wheels are located",
)

    if args.dist_dir == Path("<no-valid-dir>"):
```